content/es/community/_index.html

```diff
@@ -0,0 +1,62 @@
+---
+title: Comunidad
+layout: basic
+cid: community
+---
+
+<section id="mainContent">
+    <main>
+        <div class="content">
+            <h3>Garantizando el funcionamiento de Kubernetes para todo el mundo y en cualquier lugar.</h3>
+            <p>Conecte con la comunidad Kubernetes en nuestro canal de <a href="http://slack.k8s.io/">Slack</a>, <a href="https://discuss.kubernetes.io/">foro</a>, o únete al
+                <a href="https://groups.google.com/forum/#!forum/kubernetes-dev">Kubernetes-dev Google group</a>.
+                Cada semana se lleva a cabo una reunión de la comunidad  por videoconferencia para discutir el estado de cosas, revise el documento
+                <a href="https://github.com/kubernetes/community/blob/master/events/community-meeting.md "> Community Meeting </a> para obtener información sobre cómo participar. </p>
+            <p>También puede formar parte de la comunidad en cualquier parte del mundo a través de la
+                <a href="https://www.meetup.com/topics/kubernetes/">Comunidad de Meetups sobre Kubernets </a>o el
+                <a href="https://www.meetup.com/Kubernetes-Cloud-Native-Online-Meetup/">Meetup Online de Kubernetes Cloud Native</a>.</p>
+          </div>
+        <div class="content">
+            <h3>Special Interest Groups (SIGs)</h3>
+            <p>¿Tiene un interés especial en cómo Kubernetes trabaja con otra tecnología?
+                Consulte nuestra <a href="https://git.k8s.io/community/sig-list.md">lista de SIGs</a> que no deja de crecer:
+                Desde AWS y Openstack hasta Big Data y escalabilidad, seguro que hay un lugar dónde puede contribuir.
+                Si su interés especial no está cubierto (todavía), también encontrará instrucciones para formar un nuevo SIG.</p>
+            <p>Como miembro de la comunidad Kubernetes, le invitamos a unirse a cualquiera de las reuniones SIG. No necesita registro.</p>
+        </div>
+        <div class="content">
+          <h3>Código de Conducta</h3>
+          <p>La comunidad Kubernetes valora el respeto y la inclusión, y por ello
+              se ha cumplir un <a href="code-of-conduct/">Código de Conducta</a> en
+              cada interacción. Si observa una violación del Código de conducta
+              en un evento o reunión, en Slack, o en otro medio de comunicación,
+              puedes ponerte en contacto con el <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">Comité del Código de Conducta de Kubernetes</a>
```

Comment:
s/puedes ponerte/puede ponerse

Reply:
Fixed in https://github.com/kubernetes/website/pull/13543/commits/b94d411982f46438193abbb356901cf4a77c3fdd